tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "@edx/typescript-config",
+  "compilerOptions": {
+    "outDir": "dist",
+    "baseUrl": "./src",
+    "jsx": "react-jsx",
+    "paths": {
+      "*": ["*"]
+    }
+  },
+  "include": ["*.js", ".eslintrc.js", "src/**/*", "plugins/**/*", "jest.config.ts"],
+  "exclude": ["dist", "node_modules"],
+}
```

Comment:
Same (non blocking) question as `.eslintrc.js` and `.stylelintrc.json` - should these be MFE specific overrides or PRs to https://github.com/openedx/typescript-config?

Reply:
> unifying the configurations for all MFE repositories in a more global format

That sounds like a great plan.

> I can create a corresponding issue for this

That would be wonderful. Making issues on each of the config repos and linking them to each other to provide context would be perfect.